src/main/java/digital/slovensko/autogram/server/SignEndpoint.java

```diff
@@ -29,7 +29,7 @@ public void handle(HttpExchange exchange) throws IOException {
 
```

Comment:
Tu ešte v body.validateSigningParameters() sa podľa mňa môže používať dokument. Konkrétne pri podpisovaní už podpísaného súboru sa typ podpisu určí z dokumentu a tým pádom ho bude treba čítať už tu.

Je to celkom nepríjemná situácia. Quick-fix by bol zakázať podpisovanie zaheslovaných PDF cez API - ono to vlastne aj teraz asi vráti cez API nejaký error, že nesprávné heslo alebo čosi.

Reply:
Teraz som to skúsil, a zaheslované PDF funguje úplne v pohode cez API.

Reply:
Ak to pošlem s levelom, tak to pýta heslo, ale je tam `Odomikáte null`, čo asi nie je úplne dobre.

<img width="512" alt="Snímka obrazovky 2024-09-09 o 7 58 16" src="https://github.com/user-attachments/assets/c1ade7f8-f60c-4776-9fd8-78a266ed365c">

Ak však pošlem už podpísané PDF a nepošlem v parametroch level (čiže z request body úplne vyhodíš `parameters`), vráti to 500, lebo nevie heslo. V `SignRequestBody` vo `validateDocument()` sa volá `resolveSigningLevel()`. Tam ak nie je zadaný level, tak sa skúsi zistiť zo zadaného dokumentu.

Reply:
zajtra sa na to teda pozriem ;)

Reply:
@celuchmarek malo by to byť už vyriešené v najnovších zmenách